utils/controls-template.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style type="text/css">
+</style>
+</head>
```

Comment:
The HTML validator https://validator.w3.org/ shows multiple errors and warnings.